### PR TITLE
fix: nft detail history styling

### LINKED ISSUE #3766: NFT detail History styling 📱
History detail on NFTs needs touch up, currently looks like this:

https://user-images.githubusercontent.com/73316633/185068819-f7b91522-90f5-4fb0-b36f-04aba50e93bd.MOV

Let's make it the same as on desktop.

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3766
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

image

### CODE REVIEW

File: styles/global.scss [hunk 1]
```diff
@@ -285,3 +285,9 @@ button.button.navbar-link-background {
 .rmrk-logo {
   aspect-ratio: 127 / 42;
 }
+
+@media screen and (max-width: 768px) {
+  .b-table .table-wrapper.has-mobile-cards tr {
```
Comment (roiLeo) [inline styles/global.scss:290]:
```suggestion
  .b-table .table-wrapper.has-mobile-cards tr:not([class*="is-"]) {
```

File: styles/themes/light.scss [hunk 2]
```diff
@@ -183,6 +183,10 @@ html.light-mode {
 		}
 	}
 
+	.b-table .table-wrapper.has-mobile-cards .table {
+		background-color: #fff;
+	}
+
```
Comment (roiLeo) [inline styles/themes/light.scss:189]:
```suggestion
```
we don't need this rule

Comment (Jarsen136) [reply styles/themes/light.scss:189]: It's necessary to keep the same background color as on the pc web page.

You could find the difference from these pic.

image

image

**Comment by roiLeo** [comment]
could you check conflicts?

**Comment by Jarsen136** [comment]
> could you check conflicts?

Done

**Comment by roiLeo** [comment]
I think this already been resolved in #3742 since I can't see problems on beta 

Screenshot 2022-08-18 at 15-52-12 KodaDot - Kusama NFT Market Explorer

**Comment by Jarsen136** [comment]
> I think this already been resolved in #3742 since I can't see problems on beta
> 

Ok, Let's close this PR then.

**Pull request closed without merging**